src/impact-dashboard/ImpactDashboard.tsx

```diff
@@ -62,7 +62,7 @@ const HorizontalDivider = styled.hr`
 `;
 
 const ImpactDashboard: React.FC = () => {
-  const { countyLevelDataFailed } = useEpidemicModelState();
+  const { failed: countyLevelDataFailed } = useLocaleDataState();
```

Comment:
Sorry, newbie question:  What does it mean when we assign a key like `failed` here?  I don't see it being used anywhere so it's not clear to me why it's needed/useful?

Reply:
sorry I didn't see these comments earlier for some reason. this is just renaming the variable from `failed` to `countyLevelDataFailed`, since in the context of this component `failed` seemed like too generic a variable name

Reply:
Cool, cool.  Thanks for the explanation